src/time-slot-bar/time-slot-bar.tsx

```diff
@@ -195,17 +232,20 @@ export const TimeSlotBar = ({
             const slotWidth = TimeSlotBarHelper.calculateWidth(
                 slotStartTime,
                 slotEndTime,
-                CELL_WIDTH
+                cellWidth
             );
             const slotOffset = TimeSlotBarHelper.calculateWidth(
                 startTime,
                 slotStartTime,
-                CELL_WIDTH
+                cellWidth
             );
 
+            const isClickable = clickable && variant === "default";
+
             return (
                 <>
```

Comment:
actually this fragment is the component that needs the `key` prop instead of the `Timeslot` component

Reply:
just leaving a note because it's originally a fragment, it's possible to assign key by specifying the Fragment component like `<Fragment key={id}>`. but since using div does not break anything, I am approving the change.